docs/integrations/microsoft-azure/opentelemetry/sql-server-opentelemetry.md

```diff
@@ -1,8 +1,8 @@
 ---
-id: sql-server-opentelemetry
-title: Microsoft SQL Server - OpenTelemetry Collector
-sidebar_label: Microsoft SQL Server - OTel Collector
-description: Learn about the Sumo Logic OpenTelemetry App for Microsoft SQL Server.
+id: sql-server-linux-opentelemetry
```

Comment:
@JV0812 , you'll also need to change this file name 
<img width="899" alt="Screenshot 2023-06-01 at 8 35 38 AM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/a97cb42c-241d-48fe-9aa6-fd4eb89a6726">